### PR TITLE
workshops styles updated

### PR DESCRIPTION
**Pull request by Cocodrilette**

### CODE REVIEW

**Review by Juminstock — COMMENTED** [review]
Acepto los cambios.

File: src/components/workshop/WorkshopCard.tsx [hunk 1]
```diff
@@ -1,46 +1,79 @@
...
+
+  const Description = () => {
+    if (description === "") return null;
+    return <p>{description}</p>;
+  };
+
+  const DateEl = () => {
+    let elClassName = "w-full py-1 px-3 rounded-2xl";
+    const isToday = () =>
+      new Date(date).toDateString() === new Date().toDateString()
```
Comment (alejandro99so) [inline src/components/workshop/WorkshopCard.tsx:33]: const isToday = new Date(date).toDateString() === new Date().toDateString()

File: src/components/registrationProcess/Form.tsx [hunk 2]
```diff
@@ -76,7 +76,7 @@ export const RegistrationForm = ({
       setIsLoading(false);
       if (apiRes) {
         setIsError(false);
-        console.log({ user: apiRes.data });
+        // console.log({ user: apiRes.data });
```
Comment (alejandro99so) [inline src/components/registrationProcess/Form.tsx]: Delete comments, if you need to test you can write your custom message.

File: src/pages/api/functions/delete.ts [hunk 3]
```diff
@@ -10,7 +10,7 @@ type IResponse = {
 export const deleteUser = async (event: any, res: NextApiResponse<IResponse>) => {
     const data = event.query;
     const user = await userModel.deleteOne({ address: data.address });
-    console.log(user);
+    // console.log(user);
```
Comment (alejandro99so) [inline src/pages/api/functions/delete.ts]: deleting log

File: src/utils/RegistrationManager.ts [hunk 4]
```diff
@@ -22,7 +22,7 @@ export class RegistrationContract {
 	async isJoined(address: `0x${string}`): Promise<boolean | undefined> {
 		try {
 			const isJoined = await this.contract.isJoined(address);
-			console.log({
+			// console.log({
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting log in this file

File: src/utils/formUtils.ts [hunk 5]
```diff
@@ -159,9 +159,9 @@ export const saveUserToDB = async (userObj: UserDto): Promise<any> => {
 
 	ok = apiRes.user;
 
-	console.log({
-		apiRes,
-	});
+	// console.log({
```
Comment (alejandro99so) [inline src/utils/formUtils.ts]: deleting log in this file

File: src/utils/RegistrationManager.ts [hunk 6]
```diff
@@ -51,12 +44,10 @@ export class RegistrationContract {
...
 			const registrationFee = await this.contract.registrationFee();
 			const registrationFeeInEth = ethers.utils.formatEther(registrationFee);
-			console.log({
-				registrationFeeInEth,
-			});
+
 			return registrationFeeInEth;
 		} catch (error) {
-			console.log({ registrationFeeError: error });
+			return error;
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: `console.error("registrationFeeError: ", error)`

File: src/utils/RegistrationManager.ts [hunk 7]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
 				{ signer },
 			);
 			const success = await tx.wait();
-			console.log({ success });
+			({ success });
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting line

File: src/utils/RegistrationManager.ts [hunk 8]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 			return success;
 		} catch (error) {
-			console.log({ updatedRegistrationFeeError: error });
+			console.error(error);
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: `console.error("updatedRegistrationFeeError: ", error)`

File: src/utils/RegistrationManager.ts [hunk 9]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
-			const currentFee = await this.registrationFee();
+		try {
+			const currentFee = (await this.registrationFee()) as string;
 			if (!currentFee) return false;
 			const tx = await this.contract.connect(signer).joinIn({
 				value: ethers.utils.parseEther(currentFee),
 			});
 			const success = await tx.wait();
-			console.log({ success });
+			({ success });
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting line

File: src/utils/RegistrationManager.ts [hunk 10]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 			return success;
 		} catch (error) {
-			console.log({ joinInError: error });
+			console.error(error);
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: console.error("joinInError: ", error)

File: src/utils/RegistrationManager.ts [hunk 11]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 		}
 	}
 
 	async confirmUserQuota(address: `0x${string}`) {
 		try {
 			const tx = await this.contract.confirmUserQuota(address);
 			const success = await tx.wait();
-			console.log({ success });
+			({ success });
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting line

File: src/utils/RegistrationManager.ts [hunk 12]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 			return success;
 		} catch (error) {
-			console.log({ error });
+			console.error(error);
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: specify what is the error

File: src/utils/RegistrationManager.ts [hunk 13]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 		}
 	}
 
 	async confirmUserQuotaBatch(addresses: `0x${string}`[]) {
 		try {
 			const tx = await this.contract.confirmUserQuota(addresses);
 			const success = await tx.wait();
-			console.log({ success });
+			({ success });
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting line

File: src/utils/RegistrationManager.ts [hunk 14]
```diff
@@ -70,47 +61,47 @@ export class RegistrationContract {
...
 			return success;
 		} catch (error) {
-			console.log({ error });
+			console.error(error);
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: specify what is the error in every `console.error`

File: src/utils/RegistrationManager.ts [hunk 15]
```diff
@@ -139,14 +130,14 @@ export class RegistrationContract {
...
 		}
 	}
 
 	async getJoinedUsers(): Promise<`0x${string}`[] | undefined> {
 		try {
 			const users = await this.contract.getJoinedUsers();
-			console.log({
+			({
```
Comment (alejandro99so) [inline src/utils/RegistrationManager.ts]: deleting content of log `({users,)}`

**Review by alejandro99so — APPROVED** [review]

**Review by Juminstock — APPROVED** [review]
¡Aprobado!

**Pull request merged**